packages/flutter/lib/src/material/tabs.dart

```diff
@@ -832,6 +833,12 @@ class TabBar extends StatefulWidget implements PreferredSizeWidget {
   /// Defaults to matching platform conventions.
   final ScrollPhysics? physics;
 
+  /// Defines the [InkWell]'s `borderRadius` property.
+  ///
+  /// For example, to match the `borderRadius` of [BoxDecoration] provided as
+  /// `indicator` to this [TabBar].
```

Comment:
Instead of here, add `splashBorderRadius` in the `See also` section of the doc of `indicator`, explaining this point there.

Reply:
Remove this from here as you have added it in the see also section.